Frameworks/include/UIApplicationInternal.h

```diff
@@ -16,6 +16,9 @@
 #pragma once
 
 #import <UIKit/UIApplication.h>
+#import <UWP/WindowsFoundation.h>
```

Comment:
well you use WMSSpeechRecognitionResult so you were getting lucky with the include ordering before .... 

---
In reply to: [82294162](https://github.com/Microsoft/WinObjC/pull/1100#discussion_r82294162) [](ancestors = 82294162)